### DISCUSSION TITLE
Document sending issue

### DISCUSSION THREAD

**Discussion by SanjayasH** [OxalisCommunity/oxalis#620] [General]
Hello Team,

I am Sanjaya from Compello and I am still new to Oxalis. 

We are using oxalis-standalone and oxalis-as4 in our application to send EHF documents. 

I recently updated both oxalis-standalone and oxalis-as4 to v5.5.0 and tried to send a document to the local AP. 
Although the earlier version worked without any issues, the updated versions of both oxalis-standalone and oxalis-as4 (v5.5.0) do not send the document, causing the application to become unresponsive at certain points without any clear indication of the problem.

But the document was successfully sent once I downgraded only the oxalis-as4 to v5.4.0, .

Why is it not working when I use oxalis-standalone v5.5.0 with oxalis-as4 v5.5.0?

Please find the detailed application log here: [application-log.txt](https://github.com/OxalisCommunity/oxalis/files/11295856/application-log.txt)

I hope that someone from Oxalis Team will help me to sort this out.

Best Regards,
Sanjaya

**Comment by aaron-kumar** [comment]
@SanjayasH I am unable to reproduce your case. That means, it is working fine for us. 
Did you tried sending with Standalone client (https://github.com/OxalisCommunity/oxalis/releases/download/v5.5.0/oxalis-standalone.jar) available with v5.5.0 release? Are you facing similar problem with it?

**Comment by SanjayasH** [comment]
Yes @aaron-kumar , It worked when I sent the document directly via the Standalone client (v5.5.0). 
But I only face the specified problem when I'm using the same version along with oxalis-as4 v5.5.0 in our application.

**Comment by aaron-kumar** [comment]
@SanjayasH : I am unable to reproduce issue. 
I can see that you have some wrapper/custom integration/implementation on top of Oxalis, probably that part need investigation. Application seems to be waiting indefinitely for response and you have to find out what. Probably debugging can help to know the cause. 

As you mentioned that you are able to successfully sent message through Oxalis v5.5.0 Standalone client so it is itself says that this is Not an issue with Oxalis. 

I am changing this from issue to discussion since it is Not bug. You can continue discussion with community as part of discussion topic.

**Reply by SanjayasH** [reply]
Thank you @aaron-kumar, a conflict among few dependency versions had caused this issue and now it's working fine.
